app/queries/get_document.rb

```diff
@@ -0,0 +1,25 @@
+module Queries
+  module GetDocument
```

Comment:
I think this should be named to indicate the lock like `Queries::LockedDocument`?